web/app_prod.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use eLife\Annotations\AppKernel;
+use Symfony\Component\HttpFoundation\Request;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+$app = new AppKernel('prod');
+
+$request = Request::createFromGlobals();
+
+$response = $app->handle($request);
+$response->send();
+
+$app->terminate($request, $response);
```

Comment:
got to the last line 
<img src="http://cdn.hexjam.com/editorial_service/bases/images/000/011/631/xlarge/Untitled-1.jpg?1458208447" width="400" />

Reply:
:1st_place_medal: